### PR TITLE
KAP-1: Purpose and Guidelines for Kabuto Action Proposals

### PR DESCRIPTION
**Pull request by kenthejr**
Initial commit of KAP-1

### CODE REVIEW

File: KAP/kap-1.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+Starting with a well-vetted and refined idea and proceeding to craft a comprehensive KAP ensures that the proposal is clear, relevant, and aligns with the community's consensus and needs within the Kabuto ecosystem.
+
+### Submitting a KAP
+
+1. **Identification of Co-Authors and Editors**: Though not required, every KAP should have multiple authors or an author and multiple editors. This represents a seed of community involvement in the KAP. These participants can be cultivated from the discussions about the idea around which the KAP is written.
+
+2. **Role of a Sponsor**: The ideal scenario involves identifying a developer as a sponsor. The sponsor serves as a guide and mentor for the KAP author, providing support through the logistical aspects of the KAP process as well as providing an execution path. Being a sponsor does not preclude the individual from becoming a co-author in the future. The name of the sponsor is recorded in the "Sponsor:" field of the KAP header. An example of an ideal sponsor would be a wallet development company sponsoring a KAP written by one of its founders or partners.
```
Comment (MilanWR) [inline KAP/kap-1.md:61]: Some slight confusing wording in this paragraph I feel. A sponsor would serve as a 'guide and mentor', which feels counterintuitive with the example of an ideal sponsor (a wallet company sponsoring their own founder). Are there not two roles here? One would be a Sponsor-mentor/backer and the other would be a Sponsor-executor. Maybe there is a top level role 'Sponsor' (eg. Jason underwrites my KAP and also ensures a way for execution), and two subroles Sponsor-backer (any developer(s) who will support and guide the KAP author) and Sponsor-executor (The wallet provider who is going to implement it), whereas the both can be one and the same (where you would use the main role as sponsor).
*Role of a Sponsor**: The ideal scenario involves identifying a developer as a sponsor. The sponsor serves as a guide and mentor for the KAP author, providing support through the logistical aspects of the KAP process as well as providing an execution path. Being a sponsor does not preclude the individual from becoming a co-author in the future. The name of the sponsor is recorded in the "Sponsor:" field of the KAP header. An example of an ideal sponsor would be a wallet development company sponsoring a KAP written by one of its founders or partners.

Comment (MilanWR) [reply KAP/kap-1.md:61]: Maybe the sponsor-backers would be co-authors. But then I would remove the reference of 'guide and mentor' from the sponsor description.

**Review by MilanWR — COMMENTED** [review]
sponsor description

File: KAP/kap-1.md (hunk 1 above)
Comment (kenthejr) [reply KAP/kap-1.md:61]: So...I wanted to capture two unofficial roles: Champion & Sponsors.

Champions are the ones defining the KAP. This is likely one of the authors or co-authors.

Sponsors are the parties that would adopt or benefit from the KAP and provide feedback and "pressure" to the Champion to see it through. This party could also act as an editor, but I think the point is that a sponsor adds credibility to potential execution success.

I feel like I'm trying to put too much in too few paragraphs.

Comment (MilanWR) [reply KAP/kap-1.md:61]: yes agree, how about this (need some native english editing ;) ):

2. **Role of a Sponsor**: The ideal scenario involves identifying a end-benefiting party as a sponsor. The sponsor provides support through the logistical aspects of the KAP process as well as providing an execution path. The name of the sponsor is recorded in the "Sponsor:" field of the KAP header. An example of an ideal sponsor would be a wallet development company sponsoring a KAP about a wallet-related proposal.

Comment (kenthejr) [reply KAP/kap-1.md:61]: I think that is closer to what I'm trying to say. I'll udpate.

**Comment by kenthejr** [comment]
@MilanWR check the latest PR update. I just pushed a new commit. I think this change addresses your thoughts.

**Review by MilanWR — APPROVED** [review]

**Review by MilanWR — APPROVED** [review]

**Review by MilanWR — APPROVED** [review]
All good to go! thanks for the applied changes

**Review by MilanWR — APPROVED** [review]

**Review by MilanWR — APPROVED** [review]

**Review by MilanWR — APPROVED** [review]

**Review by MilanWR — APPROVED** [review]

**Review by kenthejr — COMMENTED** [review]
My own contributions look good to me 😝

**Review by MilanWR — APPROVED** [review]
GITHUB APPROVE DIS PLZ

File: KAP/kap-1.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+10. **Rejected Ideas**: Records ideas proposed during the KAP discussion that were ultimately rejected, along with the reasons for their rejection. This helps document the decision-making process and prevents re-proposing previously rejected ideas.
+11. **Open Issues**: A section to list ideas that surface during the draft stage, warranting further discussion. This segment ensures all issues necessary for the KAP's readiness are addressed, reducing repeated discussions.
+12. **Footnotes**: Contains cited footnotes within the KAP and serves as a repository for non-inline hyperlink targets.
+13. **Copyright/License**: Each new KAP must be released under the [Apache License Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt](https://www.apache.org/licenses/LICENSE-2.0.txt). This step ensures the open nature of the KAP and adherence to specified licensing agreements.
+
+## KAP Header Preamble
+
+Each KAP must begin with an [RFC 2822](https://datatracker.ietf.org/doc/html/rfc2822.html) style header preamble. The headers much appear in the following order. Headers marked with "*" are optional. All other headers are required.
```
Comment (BGC-ubeu) [inline KAP/kap-1.md:111]: Each KAP must begin with an [RFC 2822](https://datatracker.ietf.org/doc/html/rfc2822.html) style header preamble. The headers must appear in the following order. Headers marked with "*" are optional. All other headers are required.

**Pull request merged**